app/templates/views/check/column-errors.html

```diff
@@ -130,7 +135,10 @@ <h1 class="usa-alert__heading banner-title" data-module="track-error" data-error
   {% endcall %}
 </div>
 
-
+<!--hide the upload button and back to top link for one off sends-->
+{% if recipients.__len__() == 1 and not recipients.allowed_to_send_to and not recipients.missing_column_headers %}
```

Comment:
Same note as above regarding the method call on an object here:

```suggestion
{% if recipients|length == 1 and not recipients.allowed_to_send_to and not recipients.missing_column_headers %}
```